.github/workflows/ci.yml

```diff
@@ -0,0 +1,37 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - "main"
+      - "master"
+  pull_request:
+    branches:
+      - "main"
+      - "master"
+
+jobs:
+  unit-test:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform:
+          - ubuntu-18.04
+          - ubuntu-20.04
+        os_name:
+          - linux
+
+    runs-on: ${{ matrix.platform }}
+    env:
+      SERVER_NAME: ${{ matrix.os_name }}
```

Comment:
The `SERVER_NAME` in the APISIX is actually a hack...
Why should we bring the hack to the template?

Reply:
Thanks for pointing that out. there is no need to keep it.